### DISCUSSION TITLE
Error "java.util.zip.ZipException: duplicate entry: stamp-cert-sha256" when rebuilding APK

### DISCUSSION THREAD

**Discussion by biggestsonicfan** [iBotPeaches/Apktool#3535] [Q&A]
Greetings. I am extracting and rebuilding an APK without modification and I get the following error:
`java.util.zip.ZipException: duplicate entry: stamp-cert-sha256`

`java -jar apktool.jar d -r -s base.apk` is used to extract the APK
`java -jar apktool.jar b -c -o base-mod.apk base` is used to rebuild it as `base-mod.apk`

Flags `-r -s` are used as an example to indicate I am not modifying either, but the result is the same if I omit these flags.

**Comment by iBotPeaches** [comment]
Can I get an output of `zip -l {apk} | grep 'stamp-cert'`?

**Reply by biggestsonicfan** [reply]
Ah, I'm lacking some important info in the OP that I'm attempting this on Windows but I use Linux a lot via WSL.
`zip -l base.apk | grep 'stamp-cert'` returned nothing, as the `-l` flag for `zip` is the `compress faster` option.
`unzip -l base.apk | grep 'stamp-cert'` returned one result:
`32  1981-01-01 01:01   stamp-cert-sha256`

**Reply by iBotPeaches** [reply]
My guess is `-c` (copy original) is keeping the stamp-cert, while getting generated by something (?). Leading to that error.

**Reply by biggestsonicfan** [reply]
Just tested `java -jar apktool.jar b -o base-mod.apk base`, same error.

**Reply by iBotPeaches** [reply]
Okay, not sure then. If you can share it and fill out the bug report template. I can add it to my backlog :)

**Reply by biggestsonicfan** [reply]
Ah, actually manually searching the extracted files instead of listing the apk, `stamp-cert-sha256` is in both the `original` and `unknown` directories!
